draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -474,25 +535,30 @@
         example, a power outage to a pod can cause the Site Physical
         Availability Index to be 0% for all the routes in the pod. Partial
         fiber cut to a row of shelves can cause the Site Physical Availability
-        Index to 50% for all the routes in those shelves. The value is 0-100,
+        Index to be 50% for all the routes in those shelves. The value is 0-100,
         with 100% indicating the site is fully functional, 0% indicating the
         site is entirely out of service, and 50% indicating the site is 50%
         degraded.</t>
 
         <t>It is recommended to assign each route with one Site-ID. Depending
         on deployment, one DC can use POD number as Site-ID, another DC can
         use Row of Shelves as the Site-ID.</t>
+	<!-- XXX JMH - What happens if more than one site id is present?
+	Also, suggest making the above a "for example". -->
 
-        <t>Cloud Site/Pod failures and degradation include but are not limited
+        <t>Cloud Site/Pod failures and degradation include, but are not limited
         to, a site degradation or an entire site going down caused by a
-        variety of reasons, such as fiber cut connecting to the site or among
+        variety of reasons. Examples include fiber cuts impacting a  site or among
         pods, cooling failures, insufficient backup power, cyber threats
         attacks, too many changes outside of the maintenance window, etc.
-        Fiber-cut is not uncommon within a Cloud site or between sites.</t>
+        Fiber-cuts are not uncommon within a Cloud site or between sites.</t>
 
         <t>When those failure events happen, the edge (egress) router is
         running fine. Therefore, the ingress routers with paths to the egress
         router can't use BFD to detect the failures.</t>
+	<!-- XXX JMH - This depends on how BFD is being used.  BFD can report
+	"concatenated paths down" in some circumstances.  Please describe where
+	BFD is being used in this scenario. -->
```

Comment:
How about changing the wording to the following?

_"If the internal network attached to the egress router doesn't support BFD, it can be more challenging for the egress router to directly notify ingress routers about the physical incidents occurred within the site."_

Reply:
You're mentioning BFD here.  Your comment is that BFD can't help you at the ingress when there's issues at the egress.  BFD between ingress and egress don't help that scenario.

However, if BFD is running between from egress  to its managed resource, and BFD is running between the ingress and the egress, it's possible to do a "concatenated path down".  However, this doesn't seem to be the scenario you have in mind.

My suggestion is to either clarify how BFD is used and how it helps/doesn't help, or simply to not talk about BFD.

Reply:
My original intent was to say that the BFD session between the ingress router and egress router can't reflect the issues on the access network attached to egress router if the the BFD is not enabled on the access network attached to the egress router. You are correct that if the network attached to egress routers run the BFD, the "concatenated path down" can work.  Is there a RFC or a draft describing the "concatenated BFD sessions"? 

Can this wording address your concern? 

_"When these failure events occur, the edge (egress) router is running fine. If the internal network attached to the egress router doesn't support BFD, it can be more challenging for the egress router to directly notify ingress routers about the physical incidents within the site. However, if the access network attached to the egress router does run BFD, concatenated path down can be employed"_